### DISCUSSION TITLE
Why the same check appears several times

### DISCUSSION THREAD

**Discussion by rg-software** [WeblateOrg/weblate#6375] [Q&A]
There is a feature of Weblate I don't understand.

Suppose I translate from English to several other languages. So I have an English component set as a "source", and others are "translations". I have some line that should not be translated (say, "ID", which stays "ID" in other translations).

When I open a translation, I see the "unchanged translation" check failed. This is fine, I press "Dismiss" and go ahead.
However, when I revisit English later, I see this: "Multiple failing checks. Following checks are failing: Unchanged translation: (list of languages here)".

Why so? I have silenced these warnings in these languages, and I don't want to go through them again. Is it "by design", configurable, or I am doing something wrong here?

**Comment by nijel** [comment]
The reason behind this is that different person is looking at these. The translator dismiss the check on a translation, but the source string check is intended for the maintainer (it might be a developer or proofreader). It gives input that this string has failing check in may languages and might need either rephrasing or an ignore flag to avoid such check for all languages.
